### PR TITLE
Enable pcie_expander_bus related cases for aarch64

### PR DESCRIPTION
**Pull request by liuyd96**
libvirt support pcie_expander_bus on aarch64 since 7.7.0.
1. RHEL8 qemu doesn't support ich9 sound device on aarch64, use rng device.
2. RHEL8 qemu doesn't support host-model on aarch64, use host-passthrough.

libvirt7.6.0
```
# avocado run --vt-type libvirt --vt-arch aarch64 --vt-machine-type arm64-mmio controller.functional.positive_tests.pcie_expander_bus_numa controller.functional.positive_tests.pcie_expander_bus_busNr
JOB ID     : 4981e78c57a136ca54f37660ccd7e8bbc4f2fa09
JOB LOG    : /root/avocado/job-results/job-2021-10-28T23.30-4981e78/job.log
 (1/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_numa: CANCEL: transient disk share is not supported on current build (6.81 s)
 (2/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_busNr.default_busNr: CANCEL: transient disk share is not supported on current build (6.72 s)
 (3/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_busNr.special_busNr: CANCEL: transient disk share is not supported on current build (6.94 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 3
JOB TIME   : 21.24 s
```

libvirt7.7.0
```
# avocado run --vt-type libvirt --vt-arch aarch64 --vt-machine-type arm64-mmio controller.functional.positive_tests.pcie_expander_bus_numa controller.functional.positive_tests.pcie_expander_bus_busNr
JOB ID     : 2b51f8b47224f57808c26a01f23272b27ed4e1b7
JOB LOG    : /root/avocado/job-results/job-2021-10-28T23.30-2b51f8b/job.log
 (1/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_numa: PASS (43.52 s)
 (2/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_busNr.default_busNr: PASS (7.52 s)
 (3/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_busNr.special_busNr: PASS (12.21 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 65.29 s
```

### CODE REVIEW

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: libvirt/tests/cfg/controller/controller_functional.cfg [hunk 1]
```diff
@@ -91,7 +91,10 @@
                     # The controller attached to pcie-expander-bus should be 64:XX.X since busNr is 100(hex 64).
                     guest_patterns = "['64:.* PCI bridge: Red Hat', '64:.* PCI bridge: Intel.*82801']"
                 - pcie_expander_bus_busNr:
-                    only q35
+                    only q35, aarch64
+                    aarch64:
+                        func_supported_since_libvirt_ver = (7, 7, 0)
+                        unspported_err_msg = "pcie-expander-bus is not supported on current build"
```
[the remark this answers is not in the document]
Comment (liuyd96) [reply libvirt/tests/cfg/controller/controller_functional.cfg:97]: I verified the 'unspported_err_msg' works. The typo error should be fixed in avocado-vt
```
[yidliu@fedora avocado-vt]$ git grep unspported_err_msg
virttest/libvirt_version.py:    unspported_err_msg = params.get("unspported_err_msg",
virttest/libvirt_version.py:                LOG.error(unspported_err_msg)
virttest/libvirt_version.py:                raise exceptions.TestCancel(unspported_err_msg)
```

[the remark this answers is not in the document]
Comment (Yingshun) [reply libvirt/tests/cfg/controller/controller_functional.cfg:97]: Typo fixed in https://github.com/avocado-framework/avocado-vt/pull/3264. Could you please update your code? Thanks!

**Comment by liuyd96** [comment]
avocado-framework/avocado-vt#3264
```
# avocado run --vt-type libvirt --vt-arch aarch64 --vt-machine-type arm64-mmio controller.functional.positive_tests.pcie_expander_bus_numa controller.functional.positive_tests.pcie_expander_bus_busNr
JOB ID     : b337d3dfb11b5afd2f7aa085bae2aad6afea3543
JOB LOG    : /root/avocado/job-results/job-2021-11-03T23.41-b337d3d/job.log
 (1/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_numa: CANCEL: pcie-expander-bus is not supported on current build (7.99 s)
 (2/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_busNr.default_busNr: CANCEL: pcie-expander-bus is not supported on current build (8.06 s)
 (3/3) type_specific.io-github-autotest-libvirt.controller.functional.positive_tests.pcie_expander_bus_busNr.special_busNr: CANCEL: pcie-expander-bus is not supported on current build (7.97 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 3
JOB TIME   : 24.86 s
```

**Review by Yingshun — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**